framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -99,6 +102,7 @@
         extends AbstractNodeVisitor<TransferResult<V, S>, TransferInput<V, S>>
         implements TransferFunction<V, S> {
 
+    private final boolean useJaifInference;
```

Comment:
Please document every method and field.
Also, please put a blank line between this field and the following Javadoc comment.